python/cudf/cudf/tests/test_parquet.py

```diff
@@ -987,18 +933,13 @@ def L(list_size, first_val):
     ]
 
 
-def list_gen(
-    gen, skiprows, num_rows, lists_per_row, list_size, include_validity=False
-):
+def list_gen(gen, num_rows, lists_per_row, list_size, include_validity=False):
     """
     Generate a list column based on input parameters.
 
     Args:
         gen: A callable which generates an individual leaf element based on an
             absolute index.
-        skiprows : Generate the column as if it had started at 'skiprows'
-            instead of 0. The intent here is to emulate the skiprows
-            parameter of the parquet reader.
         num_rows : Number of rows to generate.  Again, this is to emulate the
             'num_rows' parameter of the parquet reader.
```

Comment:
The comment here needs to be updated, or perhaps the argument needs to be changed/removed. CC @nvdbaranec

Reply:
Done 👍